paddle/fluid/operators/activation_op.cc

```diff
@@ -587,6 +587,32 @@ Swish Activation Operator.
   }
 };
 
+class HardSwishOpMaker : public framework::OpProtoAndCheckerMaker {
+ public:
+  void Make() override {
+    AddInput("X", "Input of HardSwish operator");
+    AddOutput("Out", "Output of HardSwish operator");
+    AddAttr<float>("threshold", "The threshold value of Relu6")
+        .SetDefault(6.0f);
+    AddAttr<float>("scale", "The scale parameter for the input")
```

Comment:
scale parameter is the attribute of op, not the attribute of input. So it's not precise to say "parameter for the input", maybe "The scale parameter of HardSwish operator" is better

Reply:
thank you for your suggestions!